releasenotes/notes/added-parameter-ctrl_state-mcx-816dcd80e459a5ed.yaml

```diff
@@ -0,0 +1,18 @@
+---
+features:
+  - |
+    Added ctrl_state parameter to mcx() function.
```

Comment:
mcx() should be written as :func:`.QuantumCircuit.mcx()` 
also ctrl_state should be emphasized accordingly
(also in the other places in the release notes)